etc/config/cobol.amazon.properties

```diff
@@ -0,0 +1,10 @@
+compilers=&cobol
+defaultCompiler=cobol
+
+group.cobol.groupName=COBOL
+group.cobol.compilers=cobol
+group.cobol.isSemVer=true
+group.cobol.baseName=COBOL
+group.cobol.licenseName=GPLv3
+group.cobol.licenseLink=https://sourceforge.net/p/gnucobol/code/HEAD/tree/trunk/COPYING
+compiler.cobol.exe=/opt/compiler-explorer/gnucobol/bin/cobc
```

Comment:
Probably needs to use GnuCOBOL here (as mentioned for the default prop file). Group should also be 'gnucobol' as we will hopefully have at least one extra compiler in the near future (gcc)

Reply:
I highly suggest to start with two compilers, if possible, this will make those options more clear up-front.

Adding [TinyCOBOL](https://tiny-cobol.sourceforge.net/) would also be very interesting codegen wise - it generates x86 assembler instead of C. Would it be reasonable to add its final version 0.66 in this PR, too?

Reply:
If I recall correctly, TinyCOBOL is extremely outdated and it also crashes often, this is from my memory at least.

Reply:
It is what it is - not completely finished and in general only targeting COBOL85. I wouldn't use it for real-world programs but I wouldn't send a multiple KB COBOL source file of a real-world programs to godbolt either. To compare how the code generation looks and also to compare different "type" of COBOL compilers, I'd think that's a useful addition, but that's your PR, so I won't argue.
For "still a different" and much more complete compiler, what about a completely different approach with [opensource COBOL 4J](https://github.com/opensourcecobol/opensourcecobol4j), which is a spin-off from {opensource COBOL](https://github.com/opensourcecobol/opensource-cobol) to compile into java; which itself was an old Open-COBOL fork which was improved for a specific market then later partially updated from GnuCOBOL - adding the 4J or both variants?